### DISCUSSION TITLE
Webpack: local host not loading

### DISCUSSION THREAD

**Discussion by katherinevfry** [nss-evening-cohort-14/evening-client-side#100] [Help Me!] [answered]
Your Name:
Katy Fry

Have you searched the other help tickets? 
I have!

Describe your issue:
Local host not loading using npm start. It's trying to open it in the browser but I'm just getting the spinny wheel of death.

List any error messages that appear:
N/A

What have you googled? 
https://stackoverflow.com/questions/40478608/npm-start-not-starting-local-server
https://www.xspdf.com/resolution/50553719.html#:~:text=Start%20script%20missing%20error%20when,start%20command%20to%20node%20server.
https://stackoverflow.com/questions/54122014/npm-start-not-working-after-installation-of-webpack-and-babel/54122061

What I've tried:
opening a new Ubuntu window and running npm start again
sudo npm start
copying the local host link
clicking the local host link directly from my terminal

**Comment by chiestroud** [comment]
I just submitted a discussion ticket and I saw yours :) I am having the same issues.

**Reply by ajawashington** [reply]
@katherinevfry hop in Study Group 1

**Comment by katherinevfry** [comment, marked as answer]
To fix:

after trying many, many things, I restarted my computer and ran npm start again. 

Other solutions:
stop terminal.
close VS code and your terminal.
start task manager to make sure VS code and the terminal are actually closed.
restart terminal.
restart vs code.
run npm start again.

if the above doesn't work... restart your computer and try again. :)
